packages/@ember/-internals/routing/lib/system/transition.ts

```diff
@@ -139,7 +139,7 @@
  * where the router is transitioning to. It's important
  * to note that a `RouteInfo` is a linked list and this
  * property represents the leafmost route.
- * @property {RouteInfo|RouteInfoWithAttributes} to
+ * @property {?RouteInfo|RouteInfoWithAttributes} to
```

Comment:
Should we maybe say this instead:

```suggestion
 * @property {null|RouteInfo|RouteInfoWithAttributes} to
```

Reply:
I'm absolutely no expert in JSDoc but their documentation says one should use a question mark to indicate that it's nullable: https://jsdoc.app/tags-type But to be honest, I'm a little bit confused by the *non-nullable type*.

Reply:
> JSDoc but their documentation says one should use a question mark to indicate that it's nullable

Technically, we are using yuidoc (which isn't _exactly_ the same as JSDoc though they are very similar). I'm totally willing to believe that this is the right way to document it, but I find it very confusing with the question mark (does it go on both of the types? only one?).

> But to be honest, I'm a little bit confused by the non-nullable type.

Ya, same.

Reply:
I wasn't able to find any mentioning of nullable type or `?` in yuidoc docs. Maybe that's one of the differences between yuidoc and JSDoc?

Accepted your suggestion.